src/scripts/ui/inline-roll-links.ts

```diff
@@ -98,42 +98,44 @@ export const InlineRollLinks = {
             }
         });
 
-        $links.filter("[data-pf2-check]").on("click", (event) => {
-            const { pf2Check, pf2Dc, pf2Traits, pf2Label, pf2Adjustment, pf2Roller } = event.currentTarget.dataset;
-            const parent = documentFromDOM(event.currentTarget);
-            const actors = (() => {
-                if (pf2Roller === "self") {
-                    const document = documentFromDOM(event.currentTarget);
-                    const validActor = document instanceof ActorPF2e && document.canUserModify(game.user, "update");
-                    return validActor ? [document] : [];
-                }
+        for (const link of htmlQueryAll(html, "span[data-pf2-check]")) {
```

Comment:
The links are already available: you shouldn't need to requery

Reply:
The links are only used for jquery to filter and the injectRepostElement. The stuff already converted from jquery requeries. Probably because its cleaner than doing an html filter for has attribute.

If I replace for a filter I'll need to convert the other one as well, and I suspect the base way to do would be separate htmlQueries for each type of link, and then combine them together for the repost elements, because querying for every single "span" isn't quite right.

Reply:
in the interest of laziness I'll convert both, though it does make the code harder to skim now that I'm trying it.